### ISSUE TITLE
Update content on Health Corps page, 3 FAQs

### ISSUE THREAD

**Issue by abquirarte** [cagov/covid19#206]
We have been getting a ton of questions on Health Corps. We want to be able to at least address some of them on the main page.

This has already been vetted by SME (Brendan an CHHS)

Timeline- after we connect with Chaeny

### Design edit
Delete the “Register now” button after Step 5 (since it is a duplicate of step 2) and add the three FAQs below step 5

### Questions and answers

**Some healthcare professions are not listed. Are health professionals who are not listed (e.g., Medical Technicians, Lab Technicians, etc.) eligible for the California Health Corps?**

California Health Corps are currently only accepting applications for the healthcare professions listed on the website. However, we are constantly re-evaluating our needs during this fluid situation and will open the registration for other professionals, if the need arises.

**When can I expect to hear a response after completing my registration?** 
 
Currently we anticipate the process may take up to two weeks, since an incredible number of responses are received and need to prioritize certain parts of the state, where the needs are the greatest. We will begin to contact the volunteers after verification of credentials, and based on current and anticipated needs. 

**How do I return to the application after logging out?**

Visit https://healthcarevolunteers.ca.gov/ and use the account you created while applying to correct or update any information you want.

**Comment by abquirarte** [comment]
This is live!
